docs/source/reference/frame.rst

```diff
@@ -282,3 +288,4 @@ specific plotting methods of the form ``DataFrame.plot.<kind>``.
    DataFrame.plot.kde
    DataFrame.plot.density
    DataFrame.hist
+   DataFrame.kde
```

Comment:
This doesn't exist in pandas' doc. Let's remove.

Reply:
@HyukjinKwon Thanks! I removed it